### PR TITLE
Add CORE logo component

### PR DESCRIPTION
**Pull request by Joozty**
Closes https://github.com/oacore/design/issues/49

### CODE REVIEW

File: src/elements/logo/logo.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import Icon from '../icon'
+import styles from './logo.css'
+
+import { classNames } from 'utils'
+
+const Logo = React.memo(
+  ({
+    children = 'Dashboard',
```
Comment (viktor-yakubiv) [inline src/elements/logo/logo.jsx:10]:
```suggestion
    children = 'CORE',
```

File: src/elements/logo/logo.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    className,
+    src = '#core-symbol',
+    alt = 'CORE',
```
Comment (viktor-yakubiv) [inline src/elements/logo/logo.jsx:13]: I think it's better to have it editable as it is but I am not sure. What's your opinion?

File: src/elements/logo/logo.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# CORE logo
+
+
+```jsx
+<Logo>Design</Logo>
```
Comment (viktor-yakubiv) [inline src/elements/logo/logo.md:5]: Please, add an example of usage with _Link_, it's more popular to have the logo in `<a>`

File: src/elements/logo/logo.jsx (hunk 2 above)
Comment (Joozty) [reply src/elements/logo/logo.jsx:13]: It allows to use different logos, on the other hand, we apply custom negative margin on icon so any other icon can look badly 🤷‍♂️

Comment (viktor-yakubiv) [reply src/elements/logo/logo.jsx:13]: Let's keep it 🙂

**Pull request merged**